src/main/java/nl/novi/catsittermanager/repositories/FileUploadRepository.java

```diff
@@ -0,0 +1,12 @@
+package nl.novi.catsittermanager.repositories;
+
+import nl.novi.catsittermanager.models.FileUploadResponse;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface FileUploadRepository extends JpaRepository<FileUploadResponse, String > {
+    Optional<FileUploadResponse> findByFileName(String fileName);
```

Comment:
Volgens mij is deze methode niet nodig, want de fileName heeft de @Id annotation in de model FileUploadResponse. De default method findById zou de truc moeten doen. Probeer maar even.